cmd/openshift-install/migrate/azure/migrate.go

```diff
@@ -0,0 +1,45 @@
+package azure
+
+import (
+	"github.com/spf13/cobra"
+
+	azmigrate "github.com/openshift/installer/pkg/migrate/azure"
+)
+
+var (
+	azureMigrateOpts struct {
+		zone              string
+		oldResourceGroup  string
+		newResourceGroup  string
+		virtualNetwork    string
+		vnetResourceGroup string
+		link              bool
+	}
+)
+
+func runMigrateAzurePrivateDNSMigrateCmd(cmd *cobra.Command, args []string) error {
+	if (azureMigrateOpts.zone == "" || azureMigrateOpts.oldResourceGroup == "") || (azureMigrateOpts.link == true && azureMigrateOpts.virtualNetwork == "") || (azureMigrateOpts.virtualNetwork != "" && azureMigrateOpts.vnetResourceGroup == "") {
+		return cmd.Help()
+	}
```

Comment:
can we return more helpful error messages instead of just the cmd help.

Reply:
I've updated the error messages here.